src/typedefs/User.ts

```diff
@@ -0,0 +1,13 @@
+import { Address } from './Address';
+import { Company } from './Company';
+
+export interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  address: Address;
+  phone: string;
+  website: string;
+  company: Company;
+}
```

Comment:
```suggestion
export interface User {
  name: string,
  email: string,
}
```
You don't need to describe all types of object. Try to describe only fields that you will use.

Reply:
Why? Imagine if in the future this app will grow and somewhere I'll need to use more properties from User than these two.

Reply:
So you can add these properties to your interface